.travis.yml

```diff
@@ -6,11 +6,7 @@ go:
   - 1.6
 
 before_install:
-  - go get github.com/golang/mock/gomock
-  - go get github.com/golang/mock/mockgen
-  - go get github.com/onsi/gomega
-  - go get github.com/onsi/ginkgo
-  - go get golang.org/x/tools/cmd/cover
+  - make dependencies
```

Comment:
Go get should just download dependencies; can you just make this line go get -t scootdev/scoot/... ?